packages/workflow-core/src/lib/plugins/common-plugin/types.ts

```diff
@@ -75,7 +75,7 @@ export interface RiskRulesPluginParams {
       id: string;
       domain: string;
       indicator: string;
-      riskLevel: 'critical' | 'moderate' | 'positive';
+      riskLevel: 'critical' | 'high' | 'moderate' | 'positive';
```

Comment:
In general it's a good idea and can be done pretty easily by using `zod.infer`, the issue is that the schema exists in `workflows-service` but the potential use of it is in `workflow-core`. Since right now we don't share types that are related to the reports in the `common` package (something that in general we should), I don't want to move just this type, as it could lead to a bit of headache for the wide-spread of the types.